docs/2.0. APIs.md

```diff
@@ -80,26 +80,26 @@ In order to overcome shortcomings in some common libraries, the following requir
 
 #### `GET` and `HEAD` Requests
 
-- Clients performing requests using these methods SHOULD correctly handle a 301 response (moved permanently).
-- When a 301 is supported, the client MUST follow the redirect in order to retrieve the response payload.
-- Servers implementing the APIs MUST support requests using these methods to both the trailing slash and non-trailing slash path to each resource. One of these MAY produce a 301 response causing a redirect to the other if preferred.
+- Clients performing requests using these methods SHOULD correctly handle a `301` response (moved permanently).
+- When a `301` is supported, the client MUST follow the redirect in order to retrieve the response payload.
+- Servers implementing the APIs MUST support requests using these methods to both the trailing slash and non-trailing slash path to each resource. One of these MAY produce a `301` response causing a redirect to the other if preferred.
```

Comment:
```suggestion
- Clients performing requests using these methods SHOULD correctly handle a `301` (Moved Permanently) response.
- When a `301` (Moved Permanently) is supported, the client MUST follow the redirect in order to retrieve the response payload.
- Servers implementing the APIs MUST support requests using these methods to both the trailing slash and non-trailing slash path to each resource. One of these MAY produce a `301` (Moved Permanently) response causing a redirect to the other if preferred.
```

Reply:
Ah, you made the first change of these, and I wasn't sure about the repetition. (I missed these first time round because I only looked at 2, 4 and 5 codes. Oops.)